Source/g_game.c

```diff
@@ -2427,21 +2432,17 @@ void G_WorldDone(void)
 
   if (gamemapinfo)
   {
-    if (gamemapinfo->intertextsecret && secretexit)
+    if (U_CheckField(gamemapinfo->intertextsecret) && secretexit)
```

Comment:
Is this semantically equivalent? The old code did `return` if `gamemapinfo->intertextsecret` was a valid pointer, even if `gamemapinfo->intertextsecret[0]` was `'-'`. The new code only returns if both conditions are met.

Reply:
You right, fixed in https://github.com/fabiangreffrath/woof/pull/550/commits/e31abd09cd2bd68e0e18108c0bd8e5ece1345bde